internal/controller/postgrescluster/instance.go

```diff
@@ -266,6 +266,27 @@ func newObservedInstances(
 	return &observed
 }
 
+func (observed *observedInstances) writablePod(container string) (*corev1.Pod, *Instance) {
```

Comment:
:wrench: This function should have a comment.

Reply:
agreed, i'll add one in. Not sure about the name tbh, `writablePod` but it spits out a pod and instance. Couldn't think of a better one if anyone has ideas (`writablePodAndInstance` is one but not particularly good)

Reply:
:+1: 
As for the name, we have several functions in the form of `Is`_something_ (`IsRunning`, `IsPrimary`, etc). Perhaps `isWritable`? `IsWritablePod`?

Reply:
the `Is*` checks return bool values so that feels off. What about something like `GetWritablePod(container)`?

Reply:
Yeah, I could see that. Not a blocker either way.